### PR TITLE
support more xdlops mapping

### PR DESCRIPTION
**Pull request by carlushuang**

### CODE REVIEW

**Comment by shaojiewang** [comment]
LGTM

File: igemm/algo/xdlops_mapping.py [hunk 1]
```diff
@@ -261,10 +263,14 @@ def serialize(self):
         #ctrl_xdlops_mapping_t( 16 , 256,  16,  64,  2,  1,  1,  1,  1,  v_mfma_f32_16x16x1f32),     # TODO: this will fail in coalescing
 
         ctrl_xdlops_mapping_t( 128, 128,  32,  32,  4,  2,  2,  1,  1,  v_mfma_f32_16x16x1f32),
+        ctrl_xdlops_mapping_t( 128, 128,  32,  64,  4,  1,  1,  2,  1,  v_mfma_f32_32x32x1f32),
```
Comment (shaojiewang) [inline igemm/algo/xdlops_mapping.py:266]: How to distinguish the 2 different kernel? Does it require a change in kernel name?

Comment (carlushuang) [reply igemm/algo/xdlops_mapping.py:266]: yes, different name. Use different config can result in different v_mfma instruction.

**Review by shaojiewang — APPROVED** [review]

File: igemm/algo/xdlops_mapping.py [hunk 2]
```diff
@@ -243,6 +243,8 @@ def serialize(self):
         s += f"{self.inst_mfma.m}x{self.inst_mfma.n}x{self.inst_mfma.k}, " + \
                 f"lanegroup_m_tcbw:{self.lanegroup_m_per_thread()}x{self.lanegroup_m_per_cluster()}x{self.lanegroup_m_per_block()}x{self.lanegroup_m_per_wave()}, " + \
                 f"lanegroup_n_tcbw:{self.lanegroup_n_per_thread()}x{self.lanegroup_n_per_cluster()}x{self.lanegroup_n_per_block()}x{self.lanegroup_n_per_wave()}"
+        # s += "\n" + f"   lanegroup_m_per_thread:{self.lanegroup_m_per_thread()}, lanegroup_m_per_cluster:{self.lanegroup_m_per_cluster()}, lanegroup_m_per_block:{self.lanegroup_m_per_block()}, block_m_per_lanegroup:{self.block_m_per_lanegroup()}, lanegroup_m_per_wave:{self.lanegroup_m_per_wave()}"
+        # s += "\n" + f"   lanegroup_n_per_thread:{self.lanegroup_n_per_thread()}, lanegroup_n_per_cluster:{self.lanegroup_n_per_cluster()}, lanegroup_n_per_block:{self.lanegroup_n_per_block()}, block_n_per_lanegroup:{self.block_n_per_lanegroup()}, lanegroup_n_per_wave:{self.lanegroup_n_per_wave()}"
         return s
```
[the remark this answers is not in the document]
Comment (carlushuang) [reply igemm/algo/xdlops_mapping.py:248]: Sometimes need debug into the xdlops mapping...

**Pull request merged**